<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Repair</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>831</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>850</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="pb_view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ready orders</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pb_component">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Waiting for component</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pb_all">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>All orders</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_diagnostic">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Diagnostic</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout_service">
    <property name="title">
     <string>About service</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <addaction name="menuAbout_service"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="order_action"/>
   <addaction name="del_action"/>
   <addaction name="edit_action"/>
   <addaction name="cash_action"/>
   <addaction name="search_action"/>
  </widget>
  <action name="order_action">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/pngwing.com (1).png</normaloff>:/icons/pngwing.com (1).png</iconset>
   </property>
   <property name="text">
    <string>Make order</string>
   </property>
  </action>
  <action name="del_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить заказ</string>
   </property>
   <property name="toolTip">
    <string>Delete order</string>
   </property>
  </action>
  <action name="edit_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit order</string>
   </property>
  </action>
  <action name="cash_action">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/cash.png</normaloff>:/icons/cash.png</iconset>
   </property>
   <property name="text">
    <string>Cash out</string>
   </property>
  </action>
  <action name="search_action">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Search...</string>
   </property>
   <property name="toolTip">
    <string>Search for</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
